test_app/tests/rest_filters/test_utils.py

```diff
@@ -1,11 +1,25 @@
 import pytest
-from rest_framework.exceptions import ParseError
+from rest_framework.exceptions import ParseError, PermissionDenied
 
 from ansible_base.authentication.models import Authenticator
+from ansible_base.lib.abstract_models.common import AbstractCommonModel
 from ansible_base.rest_filters.utils import get_field_from_path
 
 
 def test_invalid_field_hop():
     with pytest.raises(ParseError) as excinfo:
         get_field_from_path(Authenticator, 'created_by__last_name__user')
     assert 'No related model for' in str(excinfo)
+
+
+def test_invalid_field_filter():
+    class CommonModelTest(AbstractCommonModel):
+        encrypted_fields = ["secret"]
+
+        def __init__(self):
+            self.secret = "$encrypted$"
```

Comment:
These tests run in the context of test_app which has models which likely already have encrypted_fields so we can save these lines of code and just reuse one of those models; like `test_app.models.EncryptionModel`.

Reply:
This change sounds good to me. I will make it when I am back at my workstation.